announcer.rb

```diff
@@ -23,13 +23,14 @@ module Announcer
   @interval = 5
   @query_stops = %w[71 72 73]
 
+  at_exit { remove_temp_files! }
```

Comment:
Just a heads-up that this _might_ stop working if you move to not `include`ing this module in your `Rakefile`. I'm not 100% sure.